demos/react-jsnation/tests/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "private": true,
+  "type": "module",
+  "name": "@starbeam-demos/jsnation/tests",
+  "version": "1.0.0",
+  "starbeam:type": "support:tests",
+  "dependencies": {
+    "@starbeam-demos/jsnation": "workspace:^"
+  },
+  "devDependencies": {
+    "@types/react": "^18",
+    "@testing-library/react": "^13.4.0"
+  },
+  "publishConfig": {
+    "main": "dist/index.cjs",
+    "types": "dist/index.d.ts",
+    "exports": {
+      ".": {
+        "types": "./dist/index.d.ts",
+        "import": "./dist/index.js",
+        "default": "./dist/index.cjs"
+      }
+    }
+  },
+  "scripts": {
+    "test:lint": "eslint",
+    "test:build": "tsc -b",
+    "test:types": "tsc -b"
```

Comment:
In keeping consistent with the patterns in the rest of this workspace, the scripts to run the actual specs live in the parent package, which also contains the vitest devDependencies necessary to run the specs. This package does not have scripts to run its own specs, intentionally.